tensorflow_serving/model_servers/server.cc

```diff
@@ -340,9 +342,12 @@ Status Server::BuildAndStart(const Options& server_options) {
   profiler_service_ = tensorflow::profiler::CreateProfilerService();
 
   ::grpc::ServerBuilder builder;
-  builder.AddListeningPort(
-      server_address,
-      BuildServerCredentialsFromSSLConfigFile(server_options.ssl_config_file));
+  // If defined, listen to a http port for gRPC.
+  if (server_options.grpc_port != 0) {
+    builder.AddListeningPort(
+          server_address,
+          BuildServerCredentialsFromSSLConfigFile(server_options.ssl_config_file));
```

Comment:
args are indented oddly. can you indent them 4 spaces in. that will also
take care of second arg being within the 80-columns.

Reply:
Done